docusaurus/vercel.json

```diff
@@ -31,17 +32,17 @@
       "statusCode": 301
     },
     {
-      "source": "/connector-development/config-based/",
+      "source": "/connector-development/config-based",
       "destination": "/platform/connector-development/config-based/low-code-cdk-overview",
       "statusCode": 301
     },
     {
       "source": "/project-overview/changelog",
-      "destination": "/release-notes/",
+      "destination": "/release_notes",
```

Comment:
seems funny we have one path with an underscore instead of a dash, but not world ending